src/views/view-final-confirmation.njk

```diff
@@ -33,7 +33,19 @@
         })
       }}
       
-      <p class="govuk-body"><a href="{{ Paths.CERTIFICATE_DOWNLOAD_URI }}" target="_blank" onclick="_paq.push(['trackGoal', {{ piwik.confirmationPagePDFGoalId }}]);" class="govuk-link">Download the signed copy of the application (opens in a new tab)</a>.</p>
+      <p class="govuk-body"><a id="certificate-download" href="{{ Paths.CERTIFICATE_DOWNLOAD_URI }}" target="_blank" class="govuk-link">Download the signed copy of the application (opens in a new tab)</a>.</p>
     </div>
   </div>
+  {% if piwik %}
+    <script nonce={{ nonce | dump | safe }} type="text/javascript">
```

Comment:
You can remove `type="text/javascript"`. We don't include that in our other embedded scripts